demos/interactive_face_detection_demo/cpp/detectors.cpp

```diff
@@ -268,226 +200,147 @@ void FaceDetection::fetchResults() {
             results.push_back(r);
         }
     }
+    return results;
 }
 
-AntispoofingClassifier::AntispoofingClassifier(const std::string& pathToModel,
-    const std::string& deviceForInference,
-    int maxBatch, bool isBatchDynamic, bool isAsync, bool doRawOutputMessages)
-    : BaseDetection("Antispoofing", pathToModel, deviceForInference, maxBatch, isBatchDynamic, isAsync, doRawOutputMessages),
+AntispoofingClassifier::AntispoofingClassifier(const std::string& pathToModel, bool doRawOutputMessages)
+    : BaseDetection(pathToModel, doRawOutputMessages),
     enquedFaces(0) {
 }
 
 void AntispoofingClassifier::submitRequest() {
     if (!enquedFaces)
         return;
-    if (isBatchDynamic) {
-        request->SetBatch(enquedFaces);
-    }
-    BaseDetection::submitRequest();
+    request.set_input_tensor(ov::Tensor{request.get_input_tensor(), {0, 0, 0, 0}, {enquedFaces, inShape[1], inShape[2], inShape[3]}});
```

Comment:
It's kind of tricky. You've filled input tensor in `AntispoofingClassifier::enqueue` by selecting roi's. And now you select roi with whole batch and set it as input. But if it was already filled earlier, why do we need to set it?